README.md

```diff
@@ -36,14 +36,62 @@ Every Kyma Module using an Operator follows 5 basic Principles:
 
 In case you are planning to migrate a pre-existing module within Kyma, please familiarize yourself with the [transition plan for existing modules](https://github.com/kyma-project/community/blob/main/concepts/modularization/transition.md)
 
+### Comparison to other established Frameworks
+
+#### Operator Lifecycle Manager (OLM)
+
+Compared to [OLM](https://olm.operatorframework.io/), the Kyma Modularization is similar, but distinct in a few key aspects.
+While OLM is built heavily around a static dependency expression, Kyma Modules are expected to resolve dependencies dynamically.
+
+Concretely, this means that while in OLM a Module has to declare CRDs and APIs that it depends on, in Kyma, all modules have the ability to depend on each other without declaring it in advance.
+This makes it of course harder to understand compared to a strict dependency graph, but it comes with a few key advantages:
+
+- Concurrent optimisation on controller level: every controller in Kyma is installed simultaneously and is not blocked from installation until other operators are available.
+  This makes it easy to e.g. create or configure resources that do not need to wait for the dependency (e.g. a ConfigMap can be created even before a deployment that has to wait for an API to be present).
+  While this enforces controllers to think about a case where the dependency is not present, we encourage Eventual Consistency and do not enforce a strict lifecycle model on our modules
+- Discoverability is handled not through a registry / server, but through a declarative configuration.
+  Every Module is installed through the `ModuleTemplate`, which is semantically the same as registering an operator in an OLM registry or `CatalogSource`. 
+  The ModuleTemplate however is a normal CR and can be installed into a Control-Plane dynamically and with GitOps practices. 
+  This allows multiple control-planes to offer differing modules simply at configuration time.
+  Also, we do not use File-Based Catalogs for maintaining our catalog, but maintain every `ModuleTemplate` through [Open Component Model](https://ocm.software/), an open standard to describe software artifact delivery.
+
+Regarding release channels for operators, Lifecycle Manager operates fundamentally at the same level as OLM. However, with `Kyma` we ensure bundling of the `ModuleTemplate` to a specific release channel.
+We are heavily inspired by the way that OLM handles release channels, but we do not have an intermediary `Subscription` that assigns the catalog to the channel. Instead, every module is deliverd in a `ModuleTemplate` in a channel already.
+
+Fundamentally, there is a distinct difference in parts of the `ModuleTemplate`. The ModuleTemplate contains not only a specification of the operator to be installed through OCM, but also a set of default values for a given channel when installed for the first time.
```

Comment:
```
Fundamentally, there is a distinct difference in parts of the `ModuleTemplate`.
```
To be honest, I don't understand this sentence. Perhaps it can be simplified or written more straightforward?